test_scripts/upgrade.sh

```diff
@@ -43,37 +58,40 @@ then
 
 ## Upgrade Zookeeper via reconfiguration
 echo "Upgrade Zookeeper"
-ansible-playbook --extra-vars "deployment_strategy: rolling" -i ~/.cache/molecule/platform/$SCENARIO_NAME/inventory all.yml --tags zookeeper
+ansible-playbook --extra-vars "deployment_strategy: rolling" -i ~/.cache/molecule/$MOLECULE_DIR/$SCENARIO_NAME/inventory confluent.platform.all --tags zookeeper
```

Comment:
Does `confluent.platform.all` works well for end branch < 7.x ?

Reply:
No, it won't Neeraj. It would only work for >=7.x 
My understanding is that the upgrade scripts on a branch are to test upgrades from any start_branch below it, with the end_branch being the current branch. Is that not the case? 

> There are a lot of repetitions in the code, can you please make variables for them.
I'll create aliases in the bash script. Hope that'd be fine.

Reply:
Since we are getting the start and end versions. I don't think we should be restricted on the current branch. that will give you more clean and extendable way to run any upgrade path. 
You can plan a generic jenkins job which can run any upgrade path.

Reply:
Addressed below